NeuroID/NeuroIDClass/NeuroID.swift

```diff
@@ -175,8 +177,9 @@ public enum NeuroID {
             return false
         }
 
-        NeuroID.groupAndPOST()
+        NeuroID.groupAndPOST(forceSend: true)
         NeuroID._isSDKStarted = false
+        NeuroID.linkedSiteID = nil
```

Comment:
Any chance where customer could call `startAppFlow()` and then `stop()` and then `start()`? Is the intention that if they are configured for multi-flows they will always call `startAppFlow()` and never `start()`

Reply:
There is a chance yes. But if they are configured for multi-flows they should call start/startSession/startAppFlow 
-> startAppFlow
-> startAppFlow
-> stop/stopSession